src/Microsoft.AspNetCore.SignalR.Core/IHubCallerClients`T.cs

```diff
@@ -3,12 +3,26 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A clients caller abstraction for a hub.
+    /// </summary>
+    /// <typeparam name="T">The client caller type.</typeparam>
     public interface IHubCallerClients<T> : IHubClients<T>
     {
+        /// <summary>
+        /// Gets a caller to the current caller connection.
+        /// </summary>
         T Caller { get; }
 
+        /// <summary>
+        /// Gets a caller to connections other than the current caller.
+        /// </summary>
         T Others { get; }
 
+        /// <summary>
+        /// Gets a caller to connections in a group other than the current caller.
```

Comment:
~~current caller~~ **all** connections **in the specified group, except the one which triggered the current invocation**.